src/ManageCourses.ApiClient/ManageCoursesApiClient.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Net.Http.Headers;
+
+namespace GovUk.Education.ManageCourses.ApiClient
+{
+    public partial class ManageCoursesApiClient
+    {
+        partial void PrepareRequest(System.Net.Http.HttpClient client, System.Net.Http.HttpRequestMessage request, string url)
+        {
+            var accessToken = ApiClientConfiguration.GetAccessTokenAsync().Result;
```

Comment:
`ApiClientConfiguration` vs `_apiClientConfiguration`

Reply:
Converted to a protected property (protected fields are frowned upon), which by convention is cased like this